src/requirements.txt

```diff
@@ -18,3 +18,5 @@ marshmallow
 Pillow
 netifaces
 pyqtgraph
+pytest~=6.1.0
```

Comment:
I'm not sure if we should add `pytest` to the requirements, since it is not strictly required to run Tribler.

@grimadas pointed me to [Poetry](https://python-poetry.org), which has a nice separation between main and dev dependencies. We could consider using that at a later stage.

Reply:
Make sense. 

Does the build script use `src/requirements.txt`?

Reply:
To build Tribler, we use `pyinstaller`, which automatically scans the source code for any dependency.

Reply:
Is that means, content of `src/requirements.txt` does not reflects on users, but only on devs?

I've removed this string. 
But just want to be ensure, that we are not fighting with the ghosts.

Reply:
Yes, the content of `requirements.txt` is only for devs currently.

Reply:
Got it :)